src/breedsRouter.js

```diff
@@ -89,13 +89,13 @@ breedsRouter.put('/:breedId', isBreedIdNum, checkBreedObj, (req, res, next) => {
 })
 
 // POST breed 
-breedsRouter.post('/', checkBreedObj, (req, res, next) => { // Checks keys (of req.body) are valid data type and acceptable parameters?  
+breedsRouter.post('/', checkBreedObj, (req, res, next) => { 
     const isItUpdated = breedRepository.addBreed(req.body); 
     if (isItUpdated) {
-        console.log("new breed successfully added");
-        res.status(201).send(); // 201 is NO CONTENT 
+        console.log("new breed successfully added"); 
+        res.status(200).send(isItUpdated); // 201 is NO CONTENT 200 is OK
     } else { 
-        const err = validate.generateErr400('Breed could not be added to breed database');
+        const err = validate.generateErr400('Breed could not be added to breed database'); // when would this be triggered? maybe we can get rid of this err handler. 
```

Comment:
I'd suggest keeping it in for now. The implementation of `breedRepository` could always change at a later date, and it pays to be a bit paranoid when thinking about error cases for APIs :)